xls-template.md

```diff
@@ -0,0 +1,113 @@
+---
+title: <The title is a few words, not a complete sentence>
+type: draft
+revision: 1
+description: <Description is one full (short) sentence>
+author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
```

Comment:
Can we possibly consider a way to list out authors related to specific revision numbers, like a list of revision # and the authors?  I think that would be suitable for the discussion of clawback, where @nbougalis was the author of revision 1 but someone else may be the author for revision 2

Reply:
Sure, what would you suggest?

In my opinion, it doesn't matter and isn't necessary. Readers who care about who authored individual revisions can simply look back at the Discussion/PR/commit history.

Reply:
I don't have a specific suggestion, but maybe perhaps multiple rows of 
```
---
revision: 2
author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
---
revision: 1
author: <a comma separated list of the author's or authors' name + GitHub username (in parenthesis), or name and email (in angle brackets).  Example, FirstName LastName (@GitHubUsername), FirstName LastName <foo@bar.com>, FirstName (@GitHubUsername) and GitHubUsername (@GitHubUsername)>
---
```

Reply:
Small updates do not require numbered versions, so the revision numbers have been removed. With git, it's easy to view the history of any file.